README.md

```diff
@@ -1,3 +1,5 @@
+[![codecov](https://codecov.io/gh/oraclown-test/pytelliot/branch/main/graph/badge.svg?token=jDGxaDXc9Q)](https://codecov.io/gh/oraclown-test/pytelliot)
```

Comment:
Should this be served from tellor cocecod acct?

Reply:
done